forte/sparse_ci/sparse_ci_solver.cc

```diff
@@ -661,8 +661,8 @@ bool SparseCISolver::davidson_liu_solver(const DeterminantHashVec& space,
         }
     }
 
-    if (converged == SolverStatus::NotConverged) {
-        outfile->Printf("\n  FCI did not converge!");
+    if (converged != SolverStatus::Converged) {
+        outfile->Printf("\n  The Davidson-Liu algorithm did not converge! Consider to increase DL_MAXITER. ");
```

Comment:
Here I'd suggest that we throw instead of calling `exit(1)`. 
```suggestion
        std::string msg = "\n  The Davidson-Liu algorithm did not converge! Consider increasing the option DL_MAXITER."
        throw std::runtime_error(msg);
```